src/platform/i386/Makefile

```diff
@@ -41,7 +41,8 @@ OBJS += serial.o
 OBJS += timer.o
 OBJS += chal.o
 OBJS += boot_comp.o
-OBJS += console.o
+#OBJS += console.o
```

Comment:
Console.o is also a VGA driver but I had created a new one. I can even remove it totally. What do you think? ENABLE_SERIAL is for serial.o and not this one.